docs/how-to/generate-blockchain.md

```diff
@@ -1,30 +1,19 @@
 # How to Generate Blockchain
 
 It is possible situation when blockchain is not advanced for more than `k` slots.
-In this case we should generate it. Use `cardano-block-gen` command for it.
+In this case we should generate it. Use `cardano-auxx` and command `generate-blocks` for it.
 
-Basic example of the command:
+Launch `cardano-auxx`:
 
 ```
-$ stack exec -- cardano-block-gen                \
+$ stack exec -- cardano-auxx                     \
     --system-start 0                             \
     --configuration-file node/configuration.yaml \
     --configuration-key mainnet_full             \
-    --blocks 1000                                \
-    --generated-db some-db                       \
-    --secret PATH_TO_SECRET_KEY                  \
-    --tx-count "(0,0)"                           \
-    --append
+    --db-path some-db                            \
 ```
 
-where:
-
-*  `1000` is a length of generated blockchain (number of blocks),
-*  `some-db` is a path to generated database.
-*  `PATH_TO_SECRET_KEY` is the path to a secret key we're using for generating blockchain.
-   This parameter can be specified **multiple** times. Please note that during Bootstrap era
-   we should use secret key Bootstrap stakeholders delegated heavily to.
-
-It is possible to use a custom seed for generating blocks. Use `--seed` parameter
-to define integer number that will be used as a seed, for example `--seed 100`.
-If the value of seed is not defined, random integer number will be used instead.
+Then ensure you have all the needed keys imported (see `add-key` and 
+`add-key-pool` commands). Next is running `generate-blocks <N> <seed>?` command, where
```

Comment:
This doc is mostly for DevOps, so `add-key-pool` doesn't need to be mentioned, I think.

Reply:
Let it stay, at least as a sanity check.